### PR TITLE
Self Removal From Team

### PR DESCRIPTION
**Pull request by davethenipper**
This allows any user (except the author) to remove themselves from a team they are currently on.

Tests will be added to this pull request in the future.

@Mctalian @alanlgirard @YashdalfTheGray

### CODE REVIEW

**Comment by Mctalian** [comment]
Looks good so far! Make sure you merge latest and get those tests written and you're good from my standpoint!

**Comment by YashdalfTheGray** [comment]
Approved pending the addition of tests.

**Comment by alanlgirard** [comment]
Looks good beside the whitespace you added! Approved pending tests

File: src/ideas/ideasView/ideasView.tpl.html [hunk 1]
```diff
@@ -196,13 +192,16 @@
                                     <p>You are currently not part of the team.  If you would like to become part of the team, please back the idea so the author can be notified about your interest in being involved.</p>
                                 </md-list-item>
                                 <md-list-item class="md-1-line" ng-show="!ideasView.isUserAuthor() && ideasView.isUserMemberOfTeam() && isUserLoggedIn()">
-                                    <p>You are part of the team!</p>
+                                    <p>You are part of the team!</p>
```
Comment (Mctalian) [inline src/ideas/ideasView/ideasView.tpl.html]: This is what I think @alanlgirard was talking about

Comment (YashdalfTheGray) [inline src/ideas/ideasView/ideasView.tpl.html]: That's quite a whitespace @davethenipper...you sure you're not hiding something behind that?

Comment (davethenipper) [inline src/ideas/ideasView/ideasView.tpl.html]: Lol, don't you worry about that. ;)

**Pull request merged**